Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:0.12.5
+RUN apt-get update && apt-get install -y --no-install-recommends python3-minimal python3-virtualenv python3-pip python3-dev python3-pil
+RUN python3 -m virtualenv -p /usr/bin/python3 --system-site-packages /var/www/shoop/venv
```

Comment:
Also, `venv` doesn't seem to work:

> Command '['/var/www/shoop/venv/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1

and I don't really have the inclination to figure out why when `virtualenv` does the trick :)